CoreUtilities/src/au/gov/asd/tac/constellation/utilities/text/StringUtilities.java

```diff
@@ -369,4 +369,14 @@ public static String removeSquareBracketsFromString(final String text) {
             return text;
         }
     }
+
+    /**
+     * Remove special characters from the {@link String}
+     *
+     * @param text the string to be modified.
+     * @return A {@link String} without special characters
+     */
+    public static String removeSpecialCharacters(final String text) {
+        return text != null ? text.replaceAll("[^A-Za-z0-9]", "") : null;
```

Comment:
You could use `[^\\w]` here instead; that would additionally allow underscores , but that seems reasonable to me.

Reply:
@cygnus-x-1 yeah but an _ is a special character right? I'd prefer to leave this as is if that's ok?

Reply:
@cygnus-x-1 (re-sending in case you missed it) did you prefer this still be changed or happy as is?

Reply:
I'm fine with it if you are.